.travis.yml

```diff
@@ -16,8 +16,15 @@ before_script:
   - yamllint --version
 
 script:
-  - for dir in *; do test -d "$dir" && echo "$dir" && ansible-lint "$dir"; done
-  - for dir in *; do test -d "$dir" && echo "$dir" && yamllint -c ./.testenv/yamllint.yml "$dir"; done
-  - for file in */tasks/*; do egrep -H '^- name:.*[[:upper:]]' "$file" && exit 1; done
```

Comment:
We no longer check for uppercase words in the `name` sections?

Reply:
this is will be checked with ansible-lint